javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,100 @@
+const recipeObject = {
+  id: 1,
+  title: "Gløgg",
+  picture_url: "https://upload.wikimedia.org/wikipedia/commons/thumb/e/e2/Gl%C3%B6gg_kastrull.JPG/800px-Gl%C3%B6gg_kastrull.JPG",
+  ingredients: [
+      { NAME: "Orange zest", AMOUNT: "0.5" },
+      { NAME: "Water", AMOUNT: "200 ml" },
+      { NAME: "Sugar", AMOUNT: "275 g" },
+      { NAME: "Whole cloves", AMOUNT: "5" },
+      { NAME: "Cinnamon sticks", AMOUNT: "2"},
+      { NAME: "Spice", AMOUNT: undefined },
+      { NAME: "Bottle of red wine", AMOUNT: "1"},
+      { NAME: "Raisins", AMOUNT: "100 g" },
+      { NAME: "Slipped Almonds", AMOUNT: "50 g"},
+  ],
+  description: "Mix everything, heat it, and you are good to go!"
+};
+
+const recipesDiv = document.getElementById("recipes" );
+
+function displayRecipe(recipe) {
+  const recipeDiv = document.createElement("div");
+  recipeDiv.classList.add("recipe");
+
+  const title = document.createElement("h2");
+  title.textContent = recipe.title;
+
+  const image = document.createElement("img");
+  image.src = recipe.picture_url;
+  image.alt = recipe.title;
+
+  const desc = document.createElement("p");
+  desc.textContent = recipe.description;
+
+  const ingredientsList = document.createElement("ul");
+  recipe.ingredients.forEach((ingredient) => {
+      const li = document.createElement("li");
+      li.textContent = `${ingredient.NAME}: ${ingredient.AMOUNT || "Some"}`;
```

Comment:
${ingredient.AMOUNT || "Some"} - why default is "Some"? It will show "some" when ingredient.AMOUNT is undefined. 
try "ingredient amount not specified"